docs/reference/cluster-config.txt

```diff
@@ -351,6 +351,9 @@ The master supports the following configuration settings:
 -  ``root``: Specifies the root directory of the state files. Defaults
    to ``/usr/share/determined/master``.
 
+-  ``cluster_name``: Specify a human readable name for this Determined
```

Comment:
"this cluster" ?

Reply:
Yeah, I like "this cluster" instead of "this Determined cluster"